Bleedify/Bleedify.Tests/Services/DonatieServiceTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using BleedifyModels.ModelsEF;
+using BleedifyServices;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bleedify.Tests.Services
+{
+    [TestClass]
+    public class DonatieServiceTests
+    {
+        private DonatieService _service;
+
+        [TestInitialize]
+        public void InitializeTests()
+        {
+            _service = new DonatieService();
+        }
+
+        [TestMethod]
+        public void UpdateDonatietest()
+        {
+            var donatie = new Donatie() { Id = 4, EtapaDonare = "forTestUpdate", DataDonare = DateTime.Now };
+            _service.Update(donatie);
```

Comment:
Nu e ok sa iti faci in cadrul unui test un obiect, cel mai bine e sa-l faci in Initialize si sa-l stergi in Clean-up. Pentru asta is facute Initialize si clean-up.